nextcord/ext/commands/errors.py

```diff
@@ -447,6 +448,19 @@ def __init__(self, argument: str) -> None:
         self.argument: str = argument
         super().__init__(f'Couldn\'t convert "{argument}" to PartialEmoji.')
 
+class FormattedDatetimeConversionFailure(BadArgument):
+    """Exception raised when the timestamp provided does not match the correct
+    format.
+    This inherits from :exc:`BadArgument`
```

Comment:
This will be shown already by any good IDE. Is this also mentioned by other issues inheriting from it or shall we agree on not putting this in there